websites/W/Wikiwand/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Hans5958",
+    "id": "279855717203050496"
+  },
+  "service": "Wikiwand",
+  "url": "www.wikiwand.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/8QkIyOM.png",
+  "thumbnail": "https://i.imgur.com/vLXp14J.jpg",
+  "color": "#F7F6F5",
+  "tags": ["encyclopedia", "informations", "wiki", "wikipedia"],
```

Comment:
Can you make `informations` say `information` instead. For grammatical sakes.

Reply:
Will do. FYI, the tags are based on the Wikipedia presence.